geode-lucene/src/test/java/org/apache/geode/cache/lucene/LuceneIndexMaintenanceIntegrationTest.java

```diff
@@ -307,4 +333,17 @@ public TestObject(String title, String description) {
       this.description = description;
     }
   }
+
+  private static class TestSerializer extends HeterogeneousLuceneSerializer {
+    @Override
+    public Collection<Document> toDocuments(LuceneIndex index, Object value) {
+
+      TestObject testObject = (TestObject) value;
+      if (testObject.title.equals("title 3")) {
```

Comment:
It might be nicer to pass the failed object name in the constructor, and give this serializer a clearer name, like ExceptionThrowingSerializer? That would make it clearer to someone reading the test what will happen with this serializer.